LibreHardwareMonitorLib/Hardware/Cpu/GenericCpu.cs

```diff
@@ -71,10 +74,16 @@ public GenericCpu(int processorIndex, CpuId[][] cpuId, ISettings settings) : bas
             if (_cpuLoad.IsAvailable)
             {
                 foreach (Sensor sensor in _coreLoads)
+                {
                     ActivateSensor(sensor);
+                    _loadSensorCounter++;
+                }
 
                 if (_totalLoad != null)
+                {
                     ActivateSensor(_totalLoad);
+                    _loadSensorCounter++;
```

Comment:
Just check the activated sensors for the actual count? Then we don't need this field.

Reply:
Sure, fixed